website/routes.py

```diff
@@ -513,15 +513,15 @@ def make_url_map(app):
 
         # reset password get
         Rule(
-            '/resetpassword/<verification_key>/',
+            '/resetpassword/<username>/<verification_key>/',
```

Comment:
It would be preferably if this route didn't expose a user's email address. If it's not possible to do a lookup with only a v2 key, consider instead including this in the JSON body of the request. Same for the `_post` route below.